drivers/bme280/include/bme280_internals.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2016 Kees Bakker, SODAQ
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_bme280 BME280
```

Comment:
About (not) dividing the pressure value.
If we don't divide, and keep it as Pa, It won't fit in 16 bits anymore. How do I fill in the val[] array then?